tests/visual/window-rtl.html

```diff
@@ -0,0 +1,96 @@
+<!doctype html>
+<html lang="en" class="k-typography k-no-animations">
+<head>
+    <title>Window</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" data-role="kendo-theme" href="../../packages/default/dist/all.css" />
+    <link rel="stylesheet" href="assets/styles.css" />
+    <script src="assets/scripts.js"></script>
+
+    <style>
+        #test-area {
+            grid-template-columns: repeat(3, 1fr);
+            align-items: start;
+        }
+        /* test page overrides */
+         .k-window {
+            position: relative;
+        }
+    </style>
+</head>
+<body dir="rtl" class="k-rtl">
```

Comment:
the `dir="rtl"` attribute should be removed, since jQuery are rendering only class `k-rtl` if RTL.